lua/lint/linters/pytestcov.lua

```diff
@@ -0,0 +1,106 @@
+local messages = {
+  file_not_covered = 'No test coverage was found for this file',
+  multiple_lines_not_covered = 'No test coverage found for these lines',
+  line_not_covered = 'No test coverage found for this line',
+  testing_error = 'No test coverage analysis could be performed due to errors'
+}
+
+local severities = {
+  error = vim.diagnostic.severity.ERROR,
+  info = vim.diagnostic.severity.INFO,
+}
+
+local ignored_files = {
+  "setup.py",
+  "docs/conf.py",
+  "test_.*.py",
+  "__init__.py",
+}
+
+return {
+  cmd = 'pytest',
+  stdin = false,
+  args = {"--cov-reset", "--cov-report=term-missing", "--cov=."},
```

Comment:
Would there be a way to retrieve the durrent directory of the buffer in the args? `--cov=.` would print the report for all the entire directory, when it is only needed the one where the buffer file is located